src/Arcus.Templates.AzureFunctions.Databricks.JobMetrics/Startup.cs

```diff
@@ -41,8 +42,9 @@ public override void Configure(IFunctionsHostBuilder builder)
             LoggerConfiguration logConfig = CreateLoggerConfiguration(builder);
             builder.Services.AddLogging(logging =>
             {
-                logging.AddSerilog(logConfig.CreateLogger(), dispose: true);
+                logging.RemoveMicrosoftApplicationInsightsLoggerProvider().AddSerilog(logConfig.CreateLogger(), dispose: true);
             });
+            
```

Comment:
```suggestion
                logging.RemoveMicrosoftApplicationInsightsLoggerProvider()
                       .AddSerilog(logConfig.CreateLogger(), dispose: true);
            });
```

Let's format this for readability! 👍 (others as well).

Reply:
> Let's format this for readability! 👍 (others as well).

done 👍 https://github.com/arcus-azure/arcus.templates/pull/607/commits/1ca3dbb8b640da9257d097b0b6ea0aaceef8ef38